pkg/apis/workload/v1alpha1/types.go

```diff
@@ -59,17 +60,41 @@ type ServiceProfileDescriptorSpec struct {
 	// +optional
 	BaselinePercent *int32 `json:"baselinePercent,omitempty"`
 
+	// if multiple CustomIndicator are defined, it means that we should
+	// try to satisfy all of those indicator targets
+	// +optional
+	// +listMapKey=name
+	// +listType=map
+	CustomIndicator []ServiceCustomIndicatorSpec `json:"customIndicator,omitempty"`
```

Comment:
what is the difference between `CustomIndicator` and `BusinessIndicator` actually?